netlify.toml

```
@@ -53,6 +53,11 @@
   to = "/docs/config-docker/"
   status = 301
 
+[[redirects]]
+  from = "/docs/languages/pandas"
+  to = "/docs/languages/python#pandas"
```

Redirect to canonical URL:

```suggestion
  to = "/docs/languages/python/#pandas"
```